molgenis-one-click-importer/src/main/frontend/src/components/OneClickImporter.vue

```diff
@@ -31,44 +31,42 @@
         <h5>Imports</h5>
         <ul class="imports-list list-unstyled">
 
-          <li v-for="upload in uploads" v-bind:key="upload.filename" class="upload-item">
-
-            <div v-show="upload.status === 'LOADING' ">
-              <i class="fa fa-spinner fa-pulse fa-fw "></i> {{upload.filename}}
-            </div>
+          <!-- Running job -->
+          <div v-if="job" v-show="job && job.status === 'RUNNING' || job.status === 'PENDING' ">
+            <i class="fa fa-spinner fa-pulse fa-fw "></i> Importing {{job.file}}
```

Comment:
Keeping the file name a the left edge makes the list scannable,  'Importing' is not a i18n tag. We should just keep the file name